nvidia_entrypoint.sh

```diff
@@ -82,7 +82,12 @@ fi
 echo
 
 if [[ $# -eq 0 ]]; then
-  exec "/bin/bash"
+  # If detect Vertex AI environment, launch tritonserver with supplied arguments
```

Comment:
I think this needs to be the outer conditional. Right now the new code is only running when there are no args.